commit caa640e5787045eae491c69f59fd734f5b925365 (refs/changes/36/1236/2)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2017-01-20T12:56:15-08:00 (2 years, 9 months ago)
    
    ADMINUI-2355 Modify Password guidelines to better reflect the validation rules

diff --git a/www/js/tpl/user-form.hbs b/www/js/tpl/user-form.hbs
index c8190f61..db7d12a5 100644
--- a/www/js/tpl/user-form.hbs
+++ b/www/js/tpl/user-form.hbs
@@ -37,7 +37,7 @@
                 <div class="col-xs-7">
                     <input type="password" placeholder="Enter or update password" name="password" class="form-control" autocomplete="off">
                     <a data-toggle="tooltip" data-placement="bottom" data-trigger="hover"
-                    title="Strong passwords should be at least 7-character long and contain numeric, uppercase, lowercase and non-alphabetic characters. Avoid the use of repeating or sequential characters. Do not re-use recent passwords.">Password guidelines</a>
+                    title="Passwords must contain at least one number, one uppercase letter, one lowercase letter and one special character (note: check the pwdpolicy in LDAP for the min length and allowed characters). Avoid the use of repeating or sequential characters. Do not re-use recent passwords.">Password guidelines</a>
                 </div>
             </div>
             <div class="form-group">
